From 6e927392ba04283d529ee41ae9bdb72f516ec818 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Sun, 20 Dec 2009 20:40:37 +0200
Subject: [PATCH] st.c: disable unknown key messages to stderr


Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 st.c |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/st.c b/st.c
index 52c3661..a37730a 100644
--- a/st.c
+++ b/st.c
@@ -1060,7 +1060,8 @@ kpress(XEvent *ev) {
 				/* XXX: paste X clipboard */;
 			break;
 		default:
-			fprintf(stderr, "errkey: %d\n", (int)ksym);
+		        /* No message for Ctrl, Alt, Caps etc */
+			/* fprintf(stderr, "errkey: %d\n", (int)ksym); */
 			break;
 		}
 }
-- 
1.6.5

