From: Jari Aalto <jari.aalto@cante.net>
Subject: Change st to more descriptive name stterm

---
 st.c |   12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

--- a/st.c
+++ b/st.c
@@ -2751,7 +2751,7 @@
 	}
 
 	if(!pattern)
-		die("st: can't open font %s\n", fontstr);
+		die("stterm: can't open font %s\n", fontstr);
 
 	if(fontsize > 0) {
 		FcPatternDel(pattern, FC_PIXEL_SIZE);
@@ -2775,7 +2775,7 @@
 	FcDefaultSubstitute(pattern);
 
 	if(xloadfont(&dc.font, pattern))
-		die("st: can't open font %s\n", fontstr);
+		die("stterm: can't open font %s\n", fontstr);
 
 	/* Setting character width and height. */
 	xw.cw = CEIL(dc.font.width * cwscale);
@@ -2784,17 +2784,17 @@
 	FcPatternDel(pattern, FC_SLANT);
 	FcPatternAddInteger(pattern, FC_SLANT, FC_SLANT_ITALIC);
 	if(xloadfont(&dc.ifont, pattern))
-		die("st: can't open font %s\n", fontstr);
+		die("stterm: can't open font %s\n", fontstr);
 
 	FcPatternDel(pattern, FC_WEIGHT);
 	FcPatternAddInteger(pattern, FC_WEIGHT, FC_WEIGHT_BOLD);
 	if(xloadfont(&dc.ibfont, pattern))
-		die("st: can't open font %s\n", fontstr);
+		die("stterm: can't open font %s\n", fontstr);
 
 	FcPatternDel(pattern, FC_SLANT);
 	FcPatternAddInteger(pattern, FC_SLANT, FC_SLANT_ROMAN);
 	if(xloadfont(&dc.bfont, pattern))
-		die("st: can't open font %s\n", fontstr);
+		die("stterm: can't open font %s\n", fontstr);
 
 	FcPatternDestroy(pattern);
 }
@@ -3263,7 +3263,7 @@
 
 void
 xresettitle(void) {
-	xsettitle(opt_title ? opt_title : "st");
+	xsettitle(opt_title ? opt_title : "stterm");
 }
 
 void
