From 6595a58898f96ac09efe9001f24a1c87097fc0c8 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Tue, 20 Sep 2011 12:51:52 +0300
Subject: [PATCH] st.info: rename from st to stterm
Organization: Private
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 st.info |   14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

--- a/st.info
+++ b/st.info
@@ -1,4 +1,4 @@
-st| simpleterm,
+stterm| Suckless Tools simple terminal,
 	acsc=``aaffggiijjkkllmmnnooppqqrrssttuuvvwwxxyyzz{{||}}~~,
 	am,
 	bce,
@@ -185,24 +185,24 @@
 	vpa=\E[%i%p1%dd,
 
 
-st-256color| simpleterm with 256 colors,
-	use=st,
+stterm-256color| Suckless Tools simple terminal with 256 colors,
+	use=stterm,
 	colors#256,
 	pairs#32767,
 #	Nicked from xterm-256color
 	setab=\E[%?%p1%{8}%<%t4%p1%d%e%p1%{16}%<%t10%p1%{8}%-%d%e48;5;%p1%d%;m,
 	setaf=\E[%?%p1%{8}%<%t3%p1%d%e%p1%{16}%<%t9%p1%{8}%-%d%e38;5;%p1%d%;m,
 
-st-meta| simpleterm with meta key,
-	use=st,
+stterm-meta| Suckless Tools simple terminal with meta key,
+	use=stterm,
 	km,
 	rmm=\E[?1034l,
 	smm=\E[?1034h,
 	rs2=\E[4l\E>\E[?1034h,
 	is2=\E[4l\E>\E[?1034h,
 
-st-meta-256color| simpleterm with meta key and 256 colors,
-	use=st-256color,
+stterm-meta-256color| Suckless Tools simple terminal with meta key and 256 colors,
+	use=stterm-256color,
 	km,
 	rmm=\E[?1034l,
 	smm=\E[?1034h,
